template/addon/index.html

```diff
@@ -102,10 +102,20 @@ <h1>{addon-title} v{addon-version}</h1>
             });
       })
       function copyLink(event, link) {
-          let copyLinkMsg = 'Copy Share Link to Clipboard: Ctrl+C, Enter\nPaste in the Search of the Stremio App'
-          if (window.isMobile)
-              copyLinkMsg = 'Copy Share Link to Clipboard: Long Press Text, Select Copy\nPaste in the Search of the Stremio App'
-          window.prompt(copyLinkMsg, link)
+        console.log('link', link);
+        console.log('event', event);
+    
+        const copyLinkMsg = window.isMobile ? 'The link was copied to your clipboard.\nPress and hold in the search bar of the Stremio app, select Paste and press Enter.' : 'The link was copied to your clipboard.\nPress Ctrl + V in the search bar of the Stremio app and press Enter.';
+        const promptMsg = window.isMobile ? 'Copy the link below by pressing and holding on it and selecting Copy.\nThen press and hold in the search bar of the Stremio app, select Paste and press Enter.' : 'Copy the link below by selecting it and pressing Ctrl + C.\nThen, Press Ctrl + V in the search bar of the Stremio app and press Enter.';
+    
+        navigator.clipboard.writeText(link).then(() => {
+            console.log('Link copied to clipboard');
+            window.alert(copyLinkMsg);
+        }).catch(err => {
+            console.error('Could not copy text: ', err);
+            console.log('Fallback to prompt');
+            window.prompt(promptMsg, link);
+        });
       }
```

Comment:
The other `copyLink` function in the other source file also has the following code at the end:

```
 event.preventDefault();
 return false;
```

Maybe it's missing here (not sure if even required at all).

Reply:
Yeah not sure what its for. I saw it was only present in one of the functions but wasn't sure why. It also returns false.

I just left it there in case it was needed for some reason. 

Do you want me to add it to the other function as well or remove it?

Reply:
Add it to the other function as well for now. Don't want to accidentally break something. We can clean this up later in another PR.

Reply:
I added it to the other function and it all seems to work as expected.